peps/pep-9999.rst

```diff
@@ -26,18 +25,18 @@ Motivation
 ==========
 
 Packages on PyPI often require build-time and runtime dependencies that are not
-present on PyPI. :doc:`PEP 725 <./pep-0725.rst>`` introduced metadata to express
+present on PyPI. :doc:`pep-0725` introduced metadata to express
 such dependencies. In order to make use of concrete external dependency metadata for
-a Python package, it is necessary to map given dependency identifiers to names used
-by another package manager. The same is true for PyPI package names.
+a Python package, it is necessary to map given dependency identifiers to the names
+used in other ecosystems. The same is true for PyPI package names.
```

Comment:
> The same is true for PyPI package names.

That isn't all that clear actually (my fault I'm sure). Can you read my mind here?

Reply:
Oh, I think it's the "Other key motivation" bullet point about repackaging pure Python wheels.

Reply:
Hm, now that you point me to it, I'm not sure what it means either. Are pure Python wheels relevant to the mapping?  Or are we hinting that folks could reuse the mapping schemas to map PyPI packages to their repackaged variants per-ecosystem? e.g. `dep:pypi/numpy` -> `conda-forge::numpy`?

Reply:
> Or are we hinting that folks could reuse the mapping schemas to map PyPI packages to their repackaged variants per-ecosystem?

That, or use such mappings to avoid repackaging. Either way you need that mapping I'd say. For most ecosystems it's pretty easy (typically it's `python-<pypi-name>` or some variation on that), but for conda-forge for example it's a bit of a mess.

Reply:
Ok, I moved the PyPI names sentence a bit below and rephrased that last paragraph in the section. Let me know if it's not clear enough! See https://github.com/rgommers/peps/pull/20/commits/d46f5c6c2be4e6fad47e3d83e5bf160b7203d6e9